DEVELOPMENT.md

```diff
@@ -1,6 +1,25 @@
 # Development
 
-## Running in development mode
+## Setup local database
+
+You can install PostgreSQL local https://www.postgresql.org/download/ or use Docker (see `postgres.yml`):
+```
+version: "3.2"
+services:
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_PASSWORD: pass
+    ports:
+      - 127.0.0.1:5432:5432
```

Comment:
would you please add a line, how to update `app-config.local.yaml` to make it work?